ext/gd/gd.c

```diff
@@ -2673,6 +2673,8 @@ static void php_imagepolygon(INTERNAL_FUNCTION_PARAMETERS, int filled)
 			RETURN_THROWS();
 		}
 		NPOINTS /= 2;
+	} else {
+		php_error_docref(NULL, E_DEPRECATED, "Using the $num_points parameter is deprecated");
```

Comment:
```suggestion
		php_error_docref(NULL, E_DEPRECATED, "using the $num_points parameter is deprecated");
```

This matches the usual usage and would match what the tests expect.

Reply:
No, error message are always capitalized.

Reply:
Well then I'm miss-remembering things